Hw2/hw_2.playground/Contents.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+// Сумма отдельно целых и отдельно дробных частей чисел 2.5, 9.7, 6.9, 8.2
+
+let firstNumber = 2.5
+let secondNumber = 9.7
+let thirdNumber = 6.9
+let fourthNumber = 8.2
+
+var intSum = Int(secondNumber) + Int(firstNumber) + Int(thirdNumber) + Int(fourthNumber)
```

Comment:
I can suggest you store all these `Int` values into separate variables so you won't need to cast them to `Int` again below